cmd/root.go

```diff
@@ -178,21 +158,43 @@ var rootCmd = &cobra.Command{
 
 		isBench := cmd.Use == "bench [destination]"
 
-		return Initialize(resumeJobID, isBench)
+		isMigratedToLibrary := cmd.Use == "resume [jobID]"
+
+		return Initialize(isMigratedToLibrary, isBench)
 	},
 }
 
-func Initialize(resumeJobID common.JobID, isBench bool) (err error) {
+func Initialize(isMigratedToLibrary, isBench bool) (err error) {
+	glcm.SetOutputFormat(OutputFormat)
+	glcm.SetOutputVerbosity(OutputLevel)
+
 	currPid := os.Getpid()
 	AsyncWarnMultipleProcesses(cmd.GetAzCopyAppPath(), currPid)
 	jobsAdmin.BenchmarkResults = isBench
 	Client, err = azcopy.NewClient(azcopy.ClientOptions{CapMbps: CapMbps})
 	if err != nil {
 		return err
 	}
-	Client.CurrentJobID = resumeJobID
-	if Client.CurrentJobID.IsEmpty() {
+	if !isMigratedToLibrary {
```

Comment:
This will all eventually get migrated to the appropriate commands that need this.